server/server.lua

```diff
@@ -370,8 +370,10 @@ function parseVehData(xml, fileName)
 
     vehicleInfoTable[fileName] = a
 
-	if outputLoading and outputLoading ~= nil and printDebugInformation then
-		print("Done with vehicle: " .. fileName)
+	if outputLoading and outputLoading ~= nil then
+		if printDebugInformation == nil or printDebugInformation == true then
```

Comment:
checking if `== true` is pointless for the future. I will still merge this though. Thanks for the contribution!